ui/src/reusable_ui/components/grid_sizer/GridSizer.tsx

```diff
@@ -0,0 +1,110 @@
+import React, {PureComponent} from 'react'
+import {ErrorHandling} from 'src/shared/decorators/errors'
+
+interface Props {
+  children?: JSX.Element[]
+  cellWidth?: number
+  width?: number
+}
+
+interface State {
+  columns: number
+}
+@ErrorHandling
+class CardSelectList extends PureComponent<Props, State> {
+  public static defaultProps: Partial<Props> = {
+    cellWidth: 160,
```

Comment:
maybe a default for width here? even if it is null.

Reply:
Updated